changelog.d/815.changes.rst

```diff
@@ -0,0 +1,2 @@
+Add ``__match_args__`` to generated class to support match case destructuring
+in Python 3.10. This can be controlled by ``match_args`` argument to ``@attrib.s``.
```

Comment:
```suggestion
``__match_args__`` are now generated to support Python 3.10's `Structural Pattern Matching <https://docs.python.org/3.10/whatsnew/3.10.html#pep-634-structural-pattern-matching>`_.
This can be controlled by ``match_args`` argument to the class decorators.
The argument is ignored for Python versions older than 3.10.
```